### ISSUE TITLE
Inference unseen video using pretrained model

### ISSUE THREAD

**Issue by t2kien** [HumamAlwassel/TSP#18]
Hi @HumamAlwassel, 
Thanks for your excellent work. I really appreciated it. 
I've trained your work on my own dataset. However, I am thinking about how to use trained model to inference unseen videos. Could you give me some examples that export result of a video such as action label and its start or end time.

Best regards,

**Comment by HumamAlwassel** [comment]
Hi @t2kien,

Thanks for your interest in our work.
TSP is not a localization method. It is a pertaining framework for the features used in a localization method. We demonstrate in the paper that features learned from TSP can be used with state-of-the-art localization methods such as G-TAD, P-GCN, and BMN. I recommend that you use one of these methods to train a localization model on top of the features you already learned from your dataset. All these methods have open-sourced code on GitHub. 

Hope this helps

**Comment by t2kien** [comment]
Thanks you, @HumamAlwassel. I got your points. I will try to utilize your work with other localization methods.
